cms/templates/js/show-correctness-editor.underscore

```diff
@@ -0,0 +1,48 @@
+<form>
+<h3 class="modal-section-title" id="show_correctness_label"><%- gettext('Correctness Visibility') %></h3>
+<div class="modal-section-content show-correctness">
+    <ul class="list-fields list-input" role="group" aria-labelledby="show_correctness_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="show-correctness" type="radio" value="always" aria-described-by="always_show_correctness_description">
```

Comment:
"aria-described-by" is not a valid ARIA attribute. It is "aria-describedby". You can find this sort of error using Chrome's built-in accessibility Audit (in Chrome developer tools).